spec/models/note_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+describe Note, :type => :model do
+  it 'has a valid factory' do
```

Comment:
Tried, that except it won't work without a valid default factory.  (tested that it worked though if factory is restored to allow default notable)